flake.nix

```diff
@@ -11,7 +11,7 @@
 
       flake-utils.url = "github:numtide/flake-utils";
 
-      known_folders.url = "https://github.com/ziglibs/known-folders/archive/d13ba6137084e55f873f6afb67447fe8906cc951.tar.gz";
+      known_folders.url = "https://github.com/ziglibs/known-folders/archive/220d01f3931595e3a2e2a6a0693363c0bfaf47e9.tar.gz";
```

Comment:
This seems like the wrong dep to update, can tell you were going for `tres.url`.

Reply:
Ah, sorry, I made a typo initially and fixed it, and somehow forgot to push the proper version... Wow I'm so smart.